lib/webhooks/webhooks.js

```diff
@@ -124,8 +124,20 @@ function validateRequest(authToken, twilioHeader, url, params) {
   const signatureWithoutPort = getExpectedTwilioSignature(authToken, urlWithoutPort, params);
   const validSignatureWithPort = scmp(Buffer.from(twilioHeader), Buffer.from(signatureWithPort));
   const validSignatureWithoutPort = scmp(Buffer.from(twilioHeader), Buffer.from(signatureWithoutPort));
+  const isValid = validSignatureWithoutPort || validSignatureWithPort
+  
+  if (!isValid) {
+    const searchParams = new URLSearchParams(urlObject.query)
+    const hasProjectId = searchParams.has('project_id')
+    const hasCallId = searchParams.has('call_id')
+    const hasMessageId = searchParams.has('message_id')
```

Comment:
@danieleds not sure if these cases are valid but i'm checking the query string too. Thoughts?

Reply:
@edolix I may be wrong but I don't think we ever use query strings for RELAY requests. Anyway having it won't hurt